mkdocs.yml

```diff
@@ -86,6 +86,13 @@ nav:
           - GAMS:       Documentation/Applications/gams.md
           - Gaussian:   Documentation/Applications/gaussian.md
           - Gurobi:     Documentation/Applications/gurobi.md
+          - IDAES Solvers : 
+            - Documentation/Applications/IDAES_Solvers/index.md
+            - Bonmin : Documentation/Applications/IDAES_Solvers/bonmin.md
+            - CBC : Documentation/Applications/IDAES_Solvers/cbc.md
+            - CLP : Documentation/Applications/IDAES_Solvers/clp.md
+            - Couenne : Documentation/Applications/IDAES_Solvers/couenne.md
+            - IPOPT : Documentation/Applications/IDAES_Solvers/ipopt.md
```

Comment:
I think we should consolidate these all to the same page, but then we can link to the individual sections here.

That way users know about the individual solvers available, but we don't have stub pages.